samples/msal-node-samples/ElectronSystemBrowserTestApp/public/successTemplate.html

```diff
@@ -3,23 +3,28 @@
 
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Electron system browser auth redirect page</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
```

Comment:
Do we really need to bootstrap in these pages? Since this page potentially has access to authZ code (even though PKCE provides protection), we should encourage keeping it sanitized and define a content security policy as well

Reply:
Done